cert_test.go

```diff
@@ -68,6 +76,64 @@ func TestPEM(t *testing.T) {
 	}
 }
 
+func TestSigned(t *testing.T) {
+	caCert, _, err := CA()
+	if err != nil {
+		cmd := exec.Command("mkcert", "-install")
```

Comment:
This seems to imply that running the tests will actually do the install step mentioned in the README.